### PR TITLE
Feature docker uwsgi docs cleanup

### PR DESCRIPTION
**Pull request by avoevodin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,14 +28,76 @@ cat > .env << __EOF__
...
+```
+* Create the .env file. Example:
+```shell
+cat > .env << __EOF__
+POSTGRES_DB=brigadier
+POSTGRES_USER=brigadier
+POSTGRES_PASSWORD=secret
+POSTGRES_HOST=0.0.0.0:8000
+POSTGRES_PORT=5433
+__EOF__
```
Comment (iliadmitriev) [inline README.md:100]: * it's development section, should use DJANGO_DEBUG=True
* forget about django-debug-toolbar

File: brigadier/brigadier/settings.py [hunk 2]
```diff
@@ -150,7 +150,6 @@
 ]
 
 if DEBUG and pkgutil.find_loader('debug_toolbar'):
-    import debug_toolbar
```
Comment (iliadmitriev) [inline brigadier/brigadier/settings.py:153]: why do you deleted debug_toolbar ?

**Review by iliadmitriev — CHANGES_REQUESTED** [review]
2 small fixed required

File: README.md (hunk 1 above)
Comment (avoevodin) [reply README.md:100]: tutorial for django-debug-toolbar is in another file.

File: brigadier/brigadier/settings.py (hunk 2 above)
Comment (avoevodin) [reply brigadier/brigadier/settings.py:153]: because it's not necessary import. Toolbar works without it.

**Review by iliadmitriev — APPROVED** [review]

**Pull request merged**